smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/impl/KafkaSink.java

```diff
@@ -338,6 +344,11 @@ private JsonObject extractProducerConfiguration(KafkaConnectorOutgoingConfigurat
             kafkaConfiguration.put(ProducerConfig.RECONNECT_BACKOFF_MAX_MS_CONFIG, "10000");
         }
 
+        if (!kafkaConfiguration.containsKey(ProducerConfig.DELIVERY_TIMEOUT_MS_CONFIG)) {
```

Comment:
Any reason to do this? 
Shouldn't we just keep the Kafka default?

Reply:
removed